### PR TITLE
testing/blip: new aport

### PR DESCRIPTION
**Pull request by neechbear**
https://github.com/neechbear/blip
Bash Library for Indolent Programmers
testing/

### CODE REVIEW

**Comment by Dominic-Kua** [comment]
This would be great to use in docker without having to install it by hand as it were.

**Comment by weakcamel** [comment]
👍 
The package is very lightweight but very useful, it would be fantastic to have it in aports!

File: testing/blip/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Contributor: Nicola Worthington <nicolaw@tfb.net>
+# Maintainer: Nicola Worthington <nicolaw@tfb.net>
+pkgname=blip
+pkgver=0.10
+pkgrel=0
+pkgdesc="Bash Library for Indolent Programmers"
+url="https://github.com/neechbear/blip"
```
Comment (andypost) [inline testing/blip/APKBUILD]: should lead to https://nicolaw.uk/blip

File: testing/blip/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+arch="noarch"
+license="MIT"
+depends="bash bc grep"
+makedepends="bash"
+install=""
+subpackages="$pkgname-doc"
+source="https://github.com/neechbear/blip/releases/download/v0.10-1/blip-0.10.tar.gz"
```
Comment (andypost) [inline testing/blip/APKBUILD]: Source should use kind of `https://github.com/neechbear/blip/archive/v${pkgver}.tar.gz`

File: testing/blip/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+pkgname=blip
+pkgver=0.10
+pkgrel=0
+pkgdesc="Bash Library for Indolent Programmers"
+url="https://github.com/neechbear/blip"
+arch="noarch"
+license="MIT"
+depends="bash bc grep"
+makedepends="bash"
+install=""
```
Comment (andypost) [inline testing/blip/APKBUILD]: no reason in empty definition

File: testing/blip/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+subpackages="$pkgname-doc"
+source="https://github.com/neechbear/blip/releases/download/v0.10-1/blip-0.10.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver/"
+
+build() {
+	cd "$builddir"
+}
```
Comment (andypost) [inline testing/blip/APKBUILD]: please remove no-op step

File: testing/blip/APKBUILD (hunk 2 above)
Comment (neechbear) [reply testing/blip/APKBUILD]: Release and archive downloads for this project aren't identical. I have corrected to include pkgver variables though.

**Comment by neechbear** [comment]
For some reason, Travis is showing checks as not completed, yet clicking on the link shows them passing?

**Comment by maxice8** [comment]
Merged, sorry for taking too long.

**Pull request closed without merging**